sdk/monitor/Azure.Monitor.OpenTelemetry/src/AzureMonitorOpenTelemetryOptions.cs

```diff
@@ -23,6 +24,11 @@ public class AzureMonitorOpenTelemetryOptions
         /// </remarks>
         public string ConnectionString { get; set; }
 
+        /// <summary>
+        /// Get or sets the value of <see cref="TokenCredential" />.
```

Comment:
minor, consider adding an explanation what to expect if the value is not set (e.g. if not set, iKey will be used..)